Compiler error brx.cpp

With v2.1.1, I can compile without issue. With the current master, I crash with this error:
<username>/xcfun/src/functionals/brx.cpp: In function ‘taylor<T, 1, N> BR_taylor(const T&)’:
<username>/xcfun/src/functionals/brx.cpp:54:26: error: expected ‘,’ before ‘)’ token
   static_assert(Ndeg >= 3);
                          ^
<username>/xcfun/src/functionals/brx.cpp:54:26: error: expected string-literal before ‘)’ token

Using gcc/g++ 9.3.1 on Centos7

--- a/src/functionals/brx.cpp
+++ b/src/functionals/brx.cpp
@@ -51,7 +51,7 @@ static double BR(double z) {
 // inverse of BR_y. Use linear method for simplicity.
 template <typename T, int Ndeg>
 taylor <T, 1, Ndeg> BR_taylor(const T & z0) {
-  static_assert(Ndeg >= 3);
+  static_assert(Ndeg >= 3, "Polynomial degree must be at least 3!");
 
   taylor<T, 1, Ndeg> t;
   t = 0;

